README.rst

```diff
@@ -1,6 +1,10 @@
 flatpak-docs
 ============
 
+|Build docs|
+
+.. |Build docs| image:: https://github.com/flatpak/flatpak-docs/workflows/Build%20docs/badge.svg
```

Comment:
I think we can do without silly badges everywhere.

Reply:
Done. I squashed the badge-removal commit into the previous changes.